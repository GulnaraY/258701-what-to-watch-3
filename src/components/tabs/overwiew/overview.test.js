import React from 'react';
import rerender from 'react-test-renderer';
import Overview from './overview.jsx';

const filmDetails = {
  rating: 8,
  ratingAmount: 200,
  description: `One day in sumemer`,
  director: `Tim Cook`,
  actors: [`actress`, `actor`],
};

it(`Should render Overview`, () => {
  const {rating, ratingAmount, description, director, actors} = filmDetails;
  const tree = rerender.create(
      <Overview
        rating={rating}
        ratingAmount={ratingAmount}
        description={description}
        director={director}
        actors={actors}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
