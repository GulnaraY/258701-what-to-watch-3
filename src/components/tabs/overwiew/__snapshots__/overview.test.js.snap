// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Overview 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        200
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      One day in sumemer
    </p>
    <p />
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Tim Cook
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        actress, actor
         and other
      </strong>
    </p>
  </div>,
]
`;
